modules/mergeday/manifests/install.pp

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+class mergeday::install {
+    include ::config
+    include users::buildduty
+    include packages::mercurial
+    include packages::mozilla::python27
+    include packages::mozilla::py27_virtualenv
+    include packages::mozilla::py27_mercurial
+    include packages::patch
+
+    python::virtualenv {
+        "/home/buildduty/mergeday":
```

Comment:
Nit, not blocking: I thought we're retiring this terminology in favor of `ciduty` 😄 Or better said `releaseduty`?

Reply:
We are, but I don't want to block having a proper merge day instance on unrelated work like that.

Reply:
No worries, we can ignore for now and address later on.